ws4redis/static/js/ws4redis.js

```diff
@@ -107,4 +135,33 @@ function WS4Redis(options, $) {
 		return ws.readyState;	
 	};
 	
+	function is_connecting() {
+		return ws.readyState == 0;	
```

Comment:
Suggestion accepted. Fixed that in the last commit. I had also to put another comparison to ensure that the ws variable is instantiated.

`(ws && ws.readyState ===  1)`